jaggr-core/WebContent/dojo/_loaderExt.js

```diff
@@ -280,6 +288,22 @@ combo.isI18nResource = combo.isI18nResource || function(mid) {
 	}
 };
 
+combo.addBootLayerDeps = function(deps) {
+	excludes = excludes.concat(deps);
+};
+
+/*
+ * Returns true if the specified module is defined, false otherwise
+ */
+combo.isDefined = function(name) {
+	try {
+		require(name);
+		return true;
+	} catch (ignore) {
+	}
+	return false;

```

Comment:
In that case I would probably prefer something like this (but it's completely up to you):

```js
try {
  require(name);
} catch (e) {
  return false;
}
return true;
```

It makes it look less like you're ignoring the error and more like you're handling the case of the module not being defined. Again, completely up to you.